src/unittests/floating_point.spec.ts

```diff
@@ -4434,53 +4434,72 @@ g.test('powInterval_f32')
+    { input: [-1, 0.1], expected: [reinterpretU16AsF16(0xae70), reinterpretU16AsF16(0x1400)] }, // ~[-0.1, 0]
+    { input: [1, -0.1], expected: [reinterpretU16AsF16(0x9400), reinterpretU16AsF16(0x2e70)] }, // ~[0, 0.1]
+    { input: [-1, -0.1], expected: [reinterpretU16AsF16(0xae70), reinterpretU16AsF16(0x1400)] }, // ~[-0.1, 0]
+  ] as ScalarPairToIntervalCase[],
+} as const;
 
-      // 64-bit normals
-      { input: [0, 0.1], expected: [0, 0] },
-      { input: [0, -0.1], expected: [0, 0] },
-      { input: [1, 0.1], expected: [reinterpretU32AsF32(0xb4000000), reinterpretU32AsF32(0x3dccccd8)] }, // ~[0, 0.1]
-      { input: [-1, 0.1], expected: [reinterpretU32AsF32(0xbdccccd8), reinterpretU32AsF32(0x34000000)] }, // ~[-0.1, 0]
-      { input: [1, -0.1], expected: [reinterpretU32AsF32(0xb4000000), reinterpretU32AsF32(0x3dccccd8)] }, // ~[0, 0.1]
-      { input: [-1, -0.1], expected: [reinterpretU32AsF32(0xbdccccd8), reinterpretU32AsF32(0x34000000)] }, // ~[-0.1, 0]
-
-      // Denominator out of range
-      { input: [1, kValue.f32.infinity.positive], expected: kUnboundedBounds },
-      { input: [1, kValue.f32.infinity.negative], expected: kUnboundedBounds },
-      { input: [kValue.f32.infinity.negative, kValue.f32.infinity.negative], expected: kUnboundedBounds },
-      { input: [kValue.f32.infinity.negative, kValue.f32.infinity.positive], expected: kUnboundedBounds },
-      { input: [kValue.f32.infinity.positive, kValue.f32.infinity.negative], expected: kUnboundedBounds },
-      { input: [1, kValue.f32.positive.max], expected: kUnboundedBounds },
-      { input: [1, kValue.f32.negative.min], expected: kUnboundedBounds },
-      { input: [1, 0], expected: kUnboundedBounds },
-      { input: [1, kValue.f32.subnormal.positive.max], expected: kUnboundedBounds },
-    ]
+g.test('remainderInterval')
+  .params(u =>
+    u
+      .combine('trait', ['f32', 'f16'] as const)
+      .beginSubcases()
+      .expandWithParams<ScalarPairToIntervalCase>(p => {
+        const trait = FP[p.trait];
+        const constants = trait.constants();
+        // prettier-ignore
+        return [
+          ...kRemainderCases[p.trait],
+          // Normals
+          { input: [0, 1], expected: [0, 0] },
+          { input: [0, -1], expected: [0, 0] },
+          { input: [1, 1], expected: [0, 1] },
+          { input: [1, -1], expected: [0, 1] },
+          { input: [-1, 1], expected: [-1, 0] },
+          { input: [-1, -1], expected: [-1, 0] },
+          { input: [4, 2], expected: [0, 2] },
+          { input: [-4, 2], expected: [-2, 0] },
+          { input: [4, -2], expected: [0, 2] },
+          { input: [-4, -2], expected: [-2, 0] },
+          { input: [2, 4], expected: [2, 2] },
+          { input: [-2, 4], expected: [-2, -2] },
+          { input: [2, -4], expected: [2, 2] },
+          { input: [-2, -4], expected: [-2, -2] },
+          { input: [0, 0.1], expected: [0, 0] },
+          { input: [0, -0.1], expected: [0, 0] },
```

Comment:
How about adding cases with input likes `[8.5, 2]` or `[1.125, 1]`, so that the expected interval of `trunc(x / y)` would be not as big as `1.0` and the expected interval of `x % y = x - y * trunc(x / y)` would be not as long as `y`?

Reply:
Done